src/store/traction/pacbio/plates/actions.js

```diff
@@ -18,10 +22,10 @@ const findPlate = async ({ getters }, filter) => {
   const request = getters.getPlates
   const promise = request.get({ filter, include: 'wells.requests' })
   const response = await handleResponse(promise)
-  const { success, data: { data, included = [] } = {} } = response
+  const { data, included = [] } = response.body
```

Comment:
You can do `const { success, body: { data, included = [] } } = response`

Reply:
Good spot 👍